src/main/java/org/cryptomator/integrations/keychain/KeychainAccessProvider.java

```diff
@@ -45,6 +57,18 @@
 	 */
 	void changePassphrase(String key, CharSequence passphrase) throws KeychainAccessException;
 
+	/**
+	 * Updates a passphrase with a given key and stores a name for that key. Noop, if there is no item for the given key.
+	 *
+	 * @param key         Unique key previously used while {@link #storePassphrase(String, CharSequence) storing a passphrase}.
+	 * @param displayName The according name to the key.
```

Comment:
Please specify what is the intend of this parameter. Will it be checked against the existing entry and cause a failure on mismatch? Or is it only used for the updated entry?